### DISCUSSION TITLE
Cannot build react-tiles-v2

### DISCUSSION THREAD

**Discussion by dim289** [pnp/sp-dev-fx-webparts#4788] [Q&A]
Hi,
I've tried to clone this repo to my PC and build it. 
Using node v10.24.1(nvm)

I tried to change versions in package.json.
    "@microsoft/sp-build-web": "1.11.0",
    "@microsoft/sp-tslint-rules": "1.11.0",
    "@microsoft/sp-module-interfaces": "1.11.0",
"@microsoft/sp-webpart-workbench": "1.11.0"

But still have error:
<img width="1526" alt="image" src="https://github.com/pnp/sp-dev-fx-webparts/assets/158278445/c54c5459-b662-493c-85b0-854be1e05041">

**Comment by gretchunkim** [comment]
Have you checked out the environment compatibility table for your SPFx 1.11.0? 

Your node is fine (v10), but your Typscript and React version may not be compatible. 


See link here: 
https://learn.microsoft.com/en-us/sharepoint/dev/spfx/compatibility
